Docs: test-scenarios-local.sh directory path is not correct or outdated

## Bug Report
I was going through the [Testing Guide](https://github.com/kubearmor/KubeArmor/blob/main/contribution/testing_guide.md) and found the below command in the docs.
```
Run the auto-testing framework

$ cd KubeArmor/tests
~/KubeArmor/tests$ ./test-scenarios-local.sh
```
As I checked this script exist in the `k8s_env` directory
```
raunak@DESKTOP-JDCPOL1:~$ ls KubeArmor/tests/
go.mod  go.sum  k8s_env  nonk8s_env  util
raunak@DESKTOP-JDCPOL1:~$ ls KubeArmor/tests/k8s_env/
Makefile      csp             multicontainer  scenarios                 test-scenarios-in-runtime.sh  throttling
README.md     host_scenarios  networktests    smoke                     test-scenarios-local.sh       visibility
blockposture  ksp             presets         syscalls                  tests.allow
configmap     microservices   privileged      test-scenarios-github.sh  tests.ignore
raunak@DESKTOP-JDCPOL1:~$
```
PS: (maybe not part of this issue) but when I also do run the script `test-scenarios-local.sh`
It errors with:
```
:~/KubeArmor/tests$ ./k8s_env/test-scenarios-local.sh
cat: /sys/kernel/security/lsm: No such file or directory
cat: /sys/kernel/security/lsm: No such file or directory
KubeArmor does not support native policies if AppArmor is not enabled
./k8s_env/test-scenarios-local.sh: line 594: cd: /home/raunak/KubeArmor/tests/k8s_env/../KubeArmor: No such file or directory
[DBG] Building KubeArmor
make: *** No rule to make target 'clean'.  Stop.
make: *** No targets specified and no makefile found.  Stop.
[DBG] Built KubeArmor
[INFO] Starting KubeArmor
./k8s_env/test-scenarios-local.sh: line 121: cd: /home/raunak/KubeArmor/tests/k8s_env/../deployments/CRD: No such file or directory
error: error reading [.]: recognized file extensions are [.json .yaml .yml]
[FAIL] Failed to apply
```
Also when executing 
**General Information**

- Environment description `k3s`
- Kernel version: `Linux DESKTOP-JDCPOL1 6.6.87.2-microsoft-standard-WSL2 #1 SMP PREEMPT_DYNAMIC Thu Jun  5 18:30:46 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux`
- Orchestration system version in use `kubectl version`: `1.33.3`

**Expected behavior**

Updated docs

--- a/contribution/testing_guide.md
+++ b/contribution/testing_guide.md
@@ -103,7 +103,7 @@ $ kubectl -n [namespace name] exec -it [pod name] -- bash -c [command]
 
     ```text
     $ cd KubeArmor/tests
-    ~/KubeArmor/tests$ ./test-scenarios-local.sh
+    ~/KubeArmor/tests$ ./k8s_env/test-scenarios-local.sh
     ```
 
     Check the test report
@@ -118,7 +118,7 @@ $ kubectl -n [namespace name] exec -it [pod name] -- bash -c [command]
 
     ```text
     $ cd KubeArmor/tests
-    ~/KubeArmor/tests$ ./test-scenarios-in-runtime.sh
+    ~/KubeArmor/tests$ ./k8s_env/test-scenarios-in-runtime.sh
     ```
 
     Check the test report

